src/components/Button/StyledButton.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+
+import {
+  colorWhite,
+  colorBlack,
+  fontLarge,
+  fontWeightBold,
+  borderRadius,
+  spacingSmall,
+  spacingMedium
+} from '../../styles/designTokens';
+
+const StyledButton = styled.button`
+  position: relative;
```

Comment:
![Screenshot 2019-06-13 at 08 33 07](https://user-images.githubusercontent.com/20565536/59409154-e7427500-8db5-11e9-8409-aaead11b48c4.png)
If you look closer shadow under button is rather sharp and has darker tint than main (for green it would be _colorDarkGreen_ from design tokens).
I think first you can try **border-bottom** as shadow (maybe it will round around corners nicely). If this won't work try to make shadow sharper (blur radius should be 0).